src/date-picker/README.md

```diff
@@ -12,9 +12,9 @@ Basic initialization
 ```html
 <igx-datePicker></igx-datePicker>
 ```
-Custom formatter function with passed initial date.
+Custom formatter function with passed initial date and locale.
 ```html
-<igx-datePicker [formatter]="customFormatter" [value]="dateValue">
+<igx-datePicker [formatter]="customFormatter" [value]="dateValue" [locale]="'en-US'">
```

Comment:
This is `regional` option, not `locale`. `locale` refers to language localization strings, while `regional` options refer to date and number formatting, as well as currency, etc.

Reply:
@kdinev  I'm quite sure, that the name of this property is locale, because when you pass a date or select one of the dates, it converts into string with appropriate type that you have passed, which means that this is the string type of the date.